test/runtime/gbt/tasks/callbacks/tcb-unin.precomp

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+
+$(make -f $CHPL_HOME/runtime/etc/Makefile.include \
+       CHPL_MAKE_HOME=$CHPL_HOME printcompileline) \
+  -c -o $1-c.o $1-c.c
```

Comment:
What's this file accomplishing that putting a .c into the .compopts file wouldn't?  Will we not include the appropriate -I paths to find the task.h file in that event?  (and if so, could you add a comment to that effect?)